content/overview/overview/using-cloudgov-paas.md

```diff
@@ -9,26 +9,26 @@ aliases:
   - /intro/overview/using-cloudgov-paas
 ---
 
-## Using the cloud.gov PaaS with 18F
+## Using the cloud.gov PaaS as self-service
 
-18F uses cloud.gov to develop and deploy applications in our work with agency partners. If 18F is delivering a product, the use of cloud.gov is implicit, and we recoup costs from the partner agency. This ensures the least possible friction for our teams. For example, [Every Kid in a Park](https://everykidinapark.gov/) (for the Department of the Interior) and [College Scorecard](https://collegescorecard.ed.gov/) (for the Department of Education) were both developed and launched using cloud.gov.
+- To use cloud.gov without 18F’s other delivery services, your agency needs an organization account. (As stated elsewhere, the cloud.gov team is currently [limiting access while we address scaling points]({{< relref "who-can-use-cloudgov.md" >}}).) 18F works with agencies to set up their cloud.gov presence. After that, the rest of the work is self-service. It wouldn’t require any more interaction with 18F.
 
-We delivered a presentation about [our early experience developing and using cloud.gov](http://www.altoros.com/cflive/gsa-cuts-app-deployment-from-14-months-to-2-3-days-with-cloud-foundry/) at the May 2015 Cloud Foundry conference.
+- You can then use cloud.gov to set up a number of online “spaces” required for developing a federal website. Your team could choose from a list of additional, optional applications that cloud.gov offers, including technical back-end and search capabilities. Once the requested services are available (which happens automatically upon request), you can use them from within your code.
 
-## Using the cloud.gov PaaS as self-service
+- After your team is set up on cloud.gov, the team can:
+  - Set up a Cloud Foundry “org” that correctly references the agency for billing purposes. This would include a quota.
```

Comment:
Technically we set the org up for them as part of getting them set up in general; it's not a separate step, and they can't create their own orgs at will.